ETTrace/ETTrace/EMGPerfAnalysis.mm

```diff
@@ -61,7 +71,75 @@ + (void)recordStack
     sStacksLock.unlock();
 }
 
-+ (void)setupStackRecording
++ (void)recordStackForAllThreads
+{
+    thread_act_array_t threads;
+    mach_msg_type_number_t thread_count;
+    if (task_threads(mach_task_self(), &threads, &thread_count) != KERN_SUCCESS) {
+        thread_count = 0;
+    }
+    
+    // Suspend all threads but ETTrace's
+    for (mach_msg_type_number_t i = 0; i < thread_count; i++) {
+        if (threads[i] != sETTraceThread)
```

Comment:
Usually preferable to include `{` even when the body is 1 line (like you do on line 78)